lib/util/git/index.ts

```diff
@@ -24,6 +24,7 @@ import { logger } from '../../logger';
 import { ExternalHostError } from '../../types/errors/external-host-error';
 import { GitOptions, GitProtocol } from '../../types/git';
 import { Limit, incLimitedValue } from '../../workers/global/limits';
+import * as packageCache from '../cache/package';
```

Comment:
Can we use this type of cache here?

Reply:
No

Reply:
Then, module-local cache inside `config` variable?

Reply:
It's likely to be unique due to the SHAs.. but it should instead be part of the repository cache. We should keep indefinitely a list of all base -> branch conflicted values we still need and have a routine to clean out old ones which are no longer needed.